### PR TITLE
Adding Week 2 Blog Post

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding week 2 blog post.
I have generated the docs and the blog is visible in the blog.html

msedge_WAfaxYdrwo

### CODE REVIEW

**Comment by skoudoro** [comment]
@m-agour, @xtanion ??

**Comment by m-agour** [comment]
Hello @ganimtron-10,
This post is good, brief, and does not have any typos but lacks some punctuation marks mentioned below.

File: docs/source/posts/2022/2022-06-15-week-2-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+.. post:: June 15 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+This week I had to refactor and make my code cleaner along with some bug fixing.
+I started by adding tests and tutorials so that my changes could be tested by everyone. Then I separated the mode for *selection* so that it would be easy to select an individual element and work along with it. Once the selection mode was complete I started with the *deletion* of the elements.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-15-week-2-praneeth.rst:14]: comma before `I started`

File: docs/source/posts/2022/2022-06-15-week-2-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Now as we have various modes the question arises, How do we know which mode is currently active and how would users know it?? For this, I took references from some other similar applications and came up with an idea to toggle the icon of the selected mode whenever the mode is selected/deselected as we had individual buttons with an icon for each mode.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-15-week-2-praneeth.rst:16]: Missing comma after `Now`

File: docs/source/posts/2022/2022-06-15-week-2-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+Now as we have various modes the question arises, How do we know which mode is currently active and how would users know it?? For this, I took references from some other similar applications and came up with an idea to toggle the icon of the selected mode whenever the mode is selected/deselected as we had individual buttons with an icon for each mode. 
+
+https://github.com/fury-gl/fury/pull/599
+
+.. image:: https://user-images.githubusercontent.com/64432063/174710174-87604e78-e563-458d-8db7-28941301b02c.gif
+   :width: 300
+   :height: 300
+   :align: center
+
+Along with this I also started to learn how to access key events so that in near future some keyboard shortcuts can be added.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-15-week-2-praneeth.rst:25]: Missing comma after `Along with this`

File: docs/source/posts/2022/2022-06-15-week-2-praneeth.rst [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+Did you get stuck anywhere?
+---------------------------
+No, Everything went well.
+
+
+What is coming up next?
+-----------------------
+In the current version of the DrawPanel, the shapes can be created and translated beyond the canvas boundaries so we have to clamp the positions according to the canvas.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-06-15-week-2-praneeth.rst:35]: Missing full stop or comma after `boundaries`

**Comment by ganimtron-10** [comment]
Thank you @m-agour  for the review!

**Review by xtanion — COMMENTED** [review]
Nice blog @ganimtron-10, There are no spelling or punctuation mistakes. It's good to go 👍🏽

**Pull request merged**